dev/test-studio/schema/standard/portableText/allTheBellsAndWhistles.ts

```diff
@@ -33,7 +32,7 @@ export const ptAllTheBellsAndWhistlesType = defineType({
                 type: 'object',
                 name: 'link',
                 title: 'Link',
-                components: {input: LinkAnnotationInput},
+                // components: {input: LinkAnnotationInput},
```

Comment:
This is making the preview crash

Reply:
There is something wrong with that schema in the test studio, should be removed or fixed.

Reply:
I think it's a problem in the custom component, the schema seems correct. The component has lots of "TODO" in it, that's why I decided to remove it from here.
Maybe we can remove the component entirely from the codebase, not sure the intention of that component, is 2 years old and has been in TODO since then. 

Adding this change
`  const {schemaType: type, value, onChange, onBlur, onFocus, validation} = props` 
in https://github.com/sanity-io/sanity/blob/next/dev/test-studio/schema/standard/portableText/LinkAnnotationInput.tsx#L30 will fix the issue, but it will render just all TODO boxes

Reply:
<img width="376" alt="Screenshot 2023-11-22 at 16 21 30" src="https://github.com/sanity-io/sanity/assets/46196328/271b1f30-a3a3-47cd-8e09-f1bfe4131fee">

Reply:
At some point, this was a test studio type for  `v2`, and it hasn't been properly ported over to v3 I think. Feel free to change or even remove this type.

Reply:
Removed, thanks!